packages/thirdweb/src/react/core/hooks/connection/ConnectButtonProps.ts

```diff
@@ -783,4 +783,9 @@ export type ConnectButtonProps = {
    * Refer to the [`SiweAuthOptions`](https://portal.thirdweb.com/references/typescript/v5/SiweAuthOptions) for more details
    */
   auth?: SiweAuthOptions;
+
+  /**
+   * How the OAuth login page opens, defaults to 'popup'
+   */
+  socialLoginMode?: "popup" | "redirect";
```

Comment:
Shouldn't this be an option of `inAppWallet()` instead?

Reply:
yes @gregfromstl this needs to be inside the inAppWallet props i think, would put it at the same level as where you choose your oauth login options